src/Microsoft.Dnx.TestHost.Client.Sources/TestHostWrapper.cs

```diff
@@ -105,7 +105,11 @@ public TestHostWrapper(string project, string dnx, bool debug)
                 WorkingDirectory = project,
             };
 
+#if DOTNET54 || DNXCORE50
+            Process.StartInfo.Environment["DNX_TESTHOST_TRACE"] = "1";
+#else
             Process.StartInfo.EnvironmentVariables.Add("DNX_TESTHOST_TRACE", "1");
```

Comment:
Exactly where is `EnvironmentVariables` (or `StringDictionary`) missing? [MSDN](https://msdn.microsoft.com/en-us/library/system.diagnostics.processstartinfo(v=vs.110).aspx) shows both `Environment` and `EnvironmentVariables` as portable and only mentions restrictions ("since 4.6") for `Environment`.